app/scripts/lib/channel-server-client.js

```diff
@@ -0,0 +1,294 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import a256gcm from './crypto/a256gcm';
+import base64url from 'base64url';
+import ChannelServerClientErrors from './channel-server-client-errors';
+import { DEVICE_PAIRING_CHANNEL_KEY_BYTES } from './constants';
+import { Model } from 'backbone';
+import { pick } from 'underscore';
+import hkdf from './crypto/hkdf';
+
+const CHANNEL_KEY_INFO_BUFFER = Buffer.from('identity.mozilla.com/picl/v1/pair/encryption-key', 'utf8');
```

Comment:
What provides `Buffer` for us?

Reply:
> What provides Buffer for us?

I'm going to assume it's the [Buffer implementation in node-libs-browser](https://github.com/webpack/node-libs-browser), is that the same as the one used by the crypto-relier?